build.gradle.kts

```diff
@@ -2,11 +2,24 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
-allprojects {
+plugins {
+    `maven-publish`
+}
+
+subprojects {
+    // FIXME - do we want this to be the group?
     group = "aws.sdk.kotlin"
```

Comment:
I think this group name is fine, especially since it's not being published to the public

Reply:
True that we aren't publishing for public consumption but I was thinking it might make it less confusing since we use this group name for AWS SDK for Kotlin. This is a placeholder TBD before we enable publishing to our own private repo.